actors/verifreg/src/lib.rs

```diff
@@ -675,6 +685,148 @@ impl Actor {
             data_cap_removed: removed_data_cap_amount,
         })
     }
+
+    pub fn revoke_expired_allocations<BS, RT>(
+        rt: &mut RT,
+        params: RevokeExpiredAllocationsParams,
+    ) -> Result<(), ActorError>
+    where
+        BS: Blockstore,
+        RT: Runtime<BS>,
+    {
+        // since the alloc is expired this should be safe to publically cleanup
+        rt.validate_immediate_caller_accept_any()?;
+        rt.transaction(|st: &mut State, rt| {
+            let mut allocs = MapMap::from_root(rt.store(), &st.allocations, HAMT_BIT_WIDTH, HAMT_BIT_WIDTH).context_code(ExitCode::USR_ILLEGAL_STATE, "failed to load allocations talbe")?;
+            for alloc_id in params.allocation_ids {
+                let alloc = allocs.get::<Address, AllocationID, Allocation>(params.client, alloc_id)
```

Comment:
Is it possible to type alias a `type ClaimTable = MapMap<x, y, z>` to remove the generic noise?

Reply:
I don't know how to do this since the generics are not on the type itself but rather its methods and while I tried to wire these two generic parameter sets together I could not figure out how to do so.  But I'm sure there is a way and I will look into it.

Reply:
We'll want a MapMap foreach